common-arcana.lic

```diff
@@ -149,6 +149,15 @@ module DRCA
     result == 'absorbs all of the energy'
   end
 
+  def release_invisibility
```

Comment:
This could potentially be combined with `release_cyclics` into `release_spells(property)` or `release_spells(*property)`